src/it/java/com/google/checkstyle/test/chapter4formatting/rule461verticalwhitespace/EmptyLineSeparatorTest.java

```diff
@@ -24,7 +24,7 @@ public static void setConfigurationBuilder() throws CheckstyleException, IOExcep
     @Test
     public void emptyLineSeparatorTest() throws IOException, Exception {
         
-        java.lang.Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
+        Class<EmptyLineSeparatorCheck> clazz = EmptyLineSeparatorCheck.class;
```

Comment:
Issue solved - we need to remember that Maven has different classpath than IDE - it's extended with compilable inputs - in this case `EmptyLineSeparatorInput.java`.